### ISSUE TITLE
Hi! I am having a trouble with formatting!

### ISSUE THREAD

**Issue by Urban9r33n** [Shuumatsu/racket-pretty-printer#3]
Hi, I personally have a problem with setting extension in VScode.

I am trying to use racket-pretty-printer in vscode by installing as an extension.

However, even though I set up pretty-printer as my default formatter, It did not work well.

![image](https://user-images.githubusercontent.com/40257491/98529833-77ddf700-22c1-11eb-8da3-648b7d9654ff.png)

If I type Racket-pretty as a command, it just blows up all of my codes.

![Nov-09-2020 19-30-19](https://user-images.githubusercontent.com/40257491/98530340-1bc7a280-22c2-11eb-914e-4d970d0fe7a9.gif)

I am new to using VScode, so please let me know If there is anything I should do!

Thank you!

**Comment by Shuumatsu** [comment]
currently, the parser doesn't support some of the syntaxes and comments, but I don't think this extension will clear the full text in these cases.
can you provide `super.rkt` as a gist link?

**Comment by Urban9r33n** [comment]
Thanks for your comment!

Well, I think there is no problem with my code because the extension did not work for any other codes including yours in the readme file. 

![Nov-10-2020 13-36-32](https://user-images.githubusercontent.com/40257491/98628380-cccd4c00-2359-11eb-83ee-f34ebfd4e330.gif)


I believe there is a problem with my preferences.
I must miss something while installing it.

Is there any further process required to use this extension after installing it in VSC in the extension market?

**Comment by Shuumatsu** [comment]
I just tried these steps on windows and it works:
1. upgrade stack to latest version 2.5.1
2. run `stack install` inside folder `core`
3. install the extension on a freshly installed vscode
4. create a new file named `demo.rkt` and fill it with  the code in the readme file
5. run command `Racket Pretty` in vscode

**Comment by Urban9r33n** [comment]
Thank you so much! I followed all of your steps again, and It worked when I freshly installed vscode!

I really appreciate it. This is the formatter which I really needed!

**Comment by Urban9r33n** [comment]
BTW, While I was testing I found interesting things.

Is there any reason why formatted don't work with some "check-expect" codes?

It worked well with demo codes (readme files)

![1](https://user-images.githubusercontent.com/40257491/98659730-c655c900-2387-11eb-8aec-b5d440340220.gif)


But when some codes were added, it did not work. (for some other syntax, and it did not work when there were too much of comments)

![2](https://user-images.githubusercontent.com/40257491/98659838-e5ecf180-2387-11eb-994d-5cf600a9b75a.gif)

Is this the syntax the parser doesn't support?
